src/controllers/mainController.ts

```diff
@@ -911,6 +917,27 @@ export default class MainController implements vscode.Disposable {
 		}
 	}
 
+	public async onExecutionPlan() {
+		const planContents = vscode.window.activeTextEditor.document.getText();
+		// remove file extension from name
+		let docName = vscode.window.activeTextEditor.document.fileName;
+		console.log("File name: ", docName);
```

Comment:
should we use the Logger class for debug logging, assuming we want to keep this in the code?

Reply:
I'm just going to remove them. they were more just there for while i was developing. Unless for some reason I should log them?